File: app/src/main/java/com/smallworldfs/moneytransferapp/modules/country/domain/model/CountryResponse.java
package com.smallworldfs.moneytransferapp.modules.country.domain.model;

import java.util.ArrayList;

public class CountryResponse {

    private String msg;

    private ArrayList<CountryModel> countries;

    public CountryResponse() {
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public ArrayList<CountryModel> getCountries() {
        return countries;
    }

    public void setCountries(ArrayList<CountryModel> countries) {
        this.countries = countries;
    }
}

File: app/src/main/java/com/smallworldfs/moneytransferapp/data/common/resources/country/model/CountryResponse.kt
package com.smallworldfs.moneytransferapp.data.common.resources.country.model

data class CountryResponse(val data: Data?) {
    data class Data(val countries: List<Country>)
    data class Country(val iso: String?, val name: String?, val isoPhoneCode: Int?, val logo: String?)
}
